packages/react-composites/tests/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "../dist",
```

Comment:
nit: does this need an outDir if noEmit is set to true?

Reply:
Counter-point: integration tests should be black-box tests -- if they need IDs for components inside the composite, _some way_ to get at those IDs should be part of the public API.

Making integration tests grey-box tests inevitably leads to some business logic in the system under test that is test-specific.

Reply:
Nope. Removed it.